samples/snippets/pipeline_service/create_training_pipeline_tabular_regression_sample_test.py

```diff
@@ -34,6 +34,9 @@ def teardown(teardown_training_pipeline):
 
 def test_ucaip_generated_create_training_pipeline_sample(capsys, shared_state):
 
+    # The return of the cancellation can be flaky; max of 20 runs was 215 sec.
+    shared_state["cancel_batch_prediction_job_timeout"] = 300
```

Comment:
Max execution time was 215 seconds out of 20 runs xref: https://github.com/googleapis/python-aiplatform/issues/580#issuecomment-930500944